psi4/src/psi4/libmints/mintshelper.cc

```diff
@@ -1105,6 +1171,7 @@ SharedMatrix MintsHelper::mo_eri_helper(SharedMatrix Iso, SharedMatrix C1, Share
     auto I4 = std::make_shared<Matrix>("MO ERI Tensor", nso * n1, n3 * nso);
     double **I4p = I4->pointer();
 
+#pragma omp parallel for collapse(4) num_threads(nthread_)
```

Comment:
This loop is just doing assignment. I doubt it is CPU bound. Parallellization might actually hurt performance.